sql/executor.go

```diff
@@ -146,6 +153,12 @@ func (e *Executor) execStmts(sql string, params parameters, planMaker *planner)
 			result = makeResultFromError(planMaker, err)
 		}
 		resp.Results = append(resp.Results, result)
+		// TODO(pmattis): Is this the correct time to be releasing leases acquired
+		// during execution of the statement?
+		//
+		// TODO(pmattis): Need to record the leases used by a transaction within
+		// the transaction state and restore it when the transaction is restored.
```

Comment:
This is the very next thing I'm going to work on. The TODO was more to alert reviewers that the work here is not done. Filing an issue feels unnecessary.